CI/releaseNotes.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+
+import sys
+import json
+from datetime import datetime
+import ghApiClient
+
+def allPulls(releaseDate):
+
+    result = ""
+
+    baseurl = "https://api.github.com/repos/swagger-api/swagger-petstore/pulls/"
+    content = ghApiClient.readUrl('repos/swagger-api/swagger-petstore/pulls?state=closed&base=master&per_page=100')
```

Comment:
wrong branch?


generally recheck all ref to branches not to point to master but to correct one